pkgs/tools/wayland/sirula/default.nix

```diff
@@ -8,18 +9,27 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sirula";
-  version = "1.0.0";
+  version = "unstable-2023-09-02";
```

Comment:
There's been a recent change in versioning scheme, prompting unstable versions to be prefixed with `0-`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

so  that makes it `1.0.1-unstable-2023-09-02` -- `'0-unstable...` is only when there exists no previous version.
there is a version bump in the commits from v1.0.1 to the current rev.
https://github.com/DorianRudolph/sirula/compare/v1.0.0...b15efe85ef1fe50849a33e5919d53d05f4f66090

Reply:
Ah, I was a bit quick. Yes, almost. I think it should be `1.0.0-unstable-2023-09-02`, since the current version is `1.0.0`

Reply:
> Ah, I was a bit quick. Yes, almost. I think it should be `1.0.0-unstable-2023-09-02`, since the current version is `1.0.0`

note that there is a version bump to 1.0.1 along with the update, though the git tag was not updated.

https://github.com/DorianRudolph/sirula/compare/v1.0.0...b15efe85ef1fe50849a33e5919d53d05f4f66090

Reply:
or maybe i misread the diff, idk.

Reply:
Right, no I see it too. Do you know whether that would be an official release, or whether the maintainers just prebumps the version in preparation for a release?

Reply:
https://github.com/DorianRudolph/sirula/commit/70b5ce1936c819f4e5c8674e90145e5e7badcfa8 doesn't really say much

Reply:
looking at the commit for tag v1.0.0
https://github.com/DorianRudolph/sirula/commit/24c715197a0deefe9c6b769706c3aca16eeb5f16
shows the version bump in cargo.toml from 0.0.1 -> 1.0.0

my guess is 1.0.1 but am fine with either as the tag suggests 1.0.0

Reply:
I'll open an issue for clarification upstream :)